WordPress/src/main/java/org/wordpress/android/ui/history/RevisionItemViewHolder.kt

```diff
@@ -46,13 +47,15 @@ class RevisionItemViewHolder(
             diffLayout.visibility = View.VISIBLE
 
             if (boundRevision.totalAdditions > 0) {
+                @SuppressLint("SetTextI18n")
```

Comment:
@ParaskP7 I realize this is still a draft, but this suppression caught my eye and I wanted to point out we can remove it if we instead catch `NoCredentialException`below.